.build/.kevm.rev

```diff
@@ -1 +1 @@
-f0a8d473676232c404ef3439260984505922a8e5
+aca727b80e864d08cb8e799087b900ca93195fe0
```

Comment:
https://github.com/kframework/evm-semantics/f0a8d473...aca727b8

Reply:
The purpose of `evm-symbolic.k` was to keep rules that 1) Are only needed for symbolic execution and 2) have to be regular rules, processed at kompile phase, that can generate branching.

All other rules can be moved to VSC, to a file that should have different name.

The only rule which I suspect HAVE to stay here is for split(). Can you explain what happened to it? I see `split` is still used in some places. You probably disabled those tests.

Until K guys decide where to incorporate it, I propose keeping it in a custom branch on KEVM, like we did so far. In this case the file in VSC should have different name, to avoid confusion.

Reply:
Daejun will talk to you about how to proper implement the *split* rules. The current split rule will appear neither on k-master nor on evm-master. Dwight and Everett don't like it.